lib/routes/routePUT.js

```diff
@@ -90,6 +90,14 @@ export default function routePUT(request, response, log, statsClient) {
                     return routesUtils.responseNoBody(err, null, response, 200,
                         log);
                 });
+            } else if (request.query.cors !== undefined) {
+                api.callApiMethod('bucketPutCors', request, log, err => {
+                    statsReport500(err, statsClient);
+                    // TODO: pushMetrics(err, log, utapi, 'bucketPutCors',
```

Comment:
I actually don't think we can add the pushMetric call without modifying Utapi first, I think we still need to add the action to UtapiClient method map here: 
https://github.com/scality/utapi/blob/e79a6798ecb7c363c0423e7c3d66a0fbb5a92e7d/src/lib/UtapiClient.js
Maybe the refactoring will make it so we don't have to update Integration and Utapi at the same time though? Can @bennettbuchanan can confirm what's correct?